pkg/apis/machine/v1beta1/machine_webhook.go

```diff
@@ -981,13 +981,13 @@ func validateVSphere(m *Machine, clusterID string) (bool, []string, utilerrors.A
 	errs = append(errs, validateVSphereNetwork(providerSpec.Network, field.NewPath("providerSpec", "network"))...)
 
 	if providerSpec.NumCPUs < minVSphereCPU {
-		warnings = append(warnings, fmt.Sprintf("providerSpec.numCPUs: %d is less than the minimum value (%d): the minimum value will be used instead", providerSpec.NumCPUs, minVSphereCPU))
+		warnings = append(warnings, fmt.Sprintf("providerSpec.numCPUs: %d is missing or less than the minimum value (%d): the minimum value will be used instead", providerSpec.NumCPUs, minVSphereCPU))
```

Comment:
Are we actually setting the minimum value?  Below it looks like that logic was removed from the reconciler.

Reply:
The logic was removed from the webhook, but is being done in the reconcile loop currently
https://github.com/openshift/machine-api-operator/blob/44f29cc5bc7665afe4136103836a018760952839/pkg/controller/vsphere/reconciler.go#L550-L561